### ISSUE TITLE
I can't download music in sdcard

### ISSUE THREAD

**Issue by INeedYooN** [Sangwan5688/BlackHole#733]
**Describe the bug**

**Steps To Reproduce**
3. Do this and that

**Expected behavior**

**Screenshots**

**Device Details (please complete the following information):**
 - OS: [e.g. Android, Windows, iOS, Linux]
 - Version: [e.g. Android 12, iOS 15, Windows 11 20H2]
 - Device: [e.g. Pixel 4, iPhone 13]
 - App Version [e.g. v1.13.0]

**App Logs:**

**Comment by joydeepd2020** [comment]
Yes, it is happening, after showing 100% completed the loding symbol still showing and the music didn't downloaded into SD/ INTERNAL storage.
